runtime/compiler/control/J9Options.cpp

```diff
@@ -261,6 +261,11 @@ int32_t J9::Options::_dltPostponeThreshold = 2;
 int32_t J9::Options::_expensiveCompWeight = TR::CompilationInfo::JSR292_WEIGHT;
 int32_t J9::Options::_jProfilingEnablementSampleThreshold = 10000;
 
+#if defined(J9VM_OPT_MICROJIT)
+int32_t J9::Options::_mjitEnabled = 0;
+int32_t J9::Options::_mjitInitialCount = 20;
```

Comment:
This is fine for now, but in the future, when setting counts, we should distinguish between methods with loops and methods without loops, much like the TR_JIT does.